frontend/src/bundles/ui/components/button-part/button-part.tsx

```diff
@@ -0,0 +1,189 @@
+import { ButtonSize, ButtonVariant } from '../../enums/enums';
+import { Button, CodeHighlight } from '../components';
+import styles from './styles.module.scss';
+
+const codeExample = `
+const ButtonExample: React.FC = () => {
+    return (
+        <>
+        <Button
+            variant={ButtonVariant.PRIMARY}
+            size={ButtonSize.MEDIUM}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.PRIMARY}
+            size={ButtonSize.MEDIUM}
+            disabled={true}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.SECONDARY}
+            size={ButtonSize.MEDIUM}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.SECONDARY}
+            size={ButtonSize.MEDIUM}
+            disabled={true}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.PRIMARY}
+            size={ButtonSize.SMALL}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.PRIMARY}
+            size={ButtonSize.SMALL}
+            disabled={true}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.SECONDARY}
+            size={ButtonSize.SMALL}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.SECONDARY}
+            size={ButtonSize.SMALL}
+            disabled={true}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button variant={ButtonVariant.PLAIN} size={ButtonSize.SMALL}>
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button
+            variant={ButtonVariant.PLAIN}
+            size={ButtonSize.SMALL}
+            disabled={true}
+        >
+            <span>+</span>
+            <span>Button</span>
+            <span>˅</span>
+        </Button>
+        <Button variant={ButtonVariant.ROUND}>+</Button>
+        </>
+    );
+}
+
+export { ButtonExample }
+`;
+
+const ButtonPart: React.FC = () => {
+    return (
+        <div className={styles.container}>
```

Comment:
it is better to wrap in a block with a global class a container that is scaffolding